installing/installing-preparing.adoc

```diff
@@ -44,9 +44,9 @@ If you want to use {product-title} but do not want to manage the cluster yoursel
 [id="installing-preparing-migrate"]
 === Have you used {product-title} 3 and want to use {product-title} 4?
 
-If you used {product-title} 3 and want to try {product-title} 4, you need to understand how different {product-title} 4 is. {product-title} 4 weaves the Operators that package, deploy, and manage Kubernetes applications and the operating system that the platform runs on, {op-system-first}, together seamlessly. Instead of deploying machines and configuring their operating systems so that you can install {product-title} on them, the {op-system} operating system is an integral part of the {product-title} cluster. Deploying the operating system for the cluster machines as part of the installation process for {product-title}. See xref:../migrating_from_ocp_3_to_4/planning-migration-3-4.adoc#migration-comparing-ocp-3-4[Comparing OpenShift Container Platform 3 and OpenShift Container Platform 4].
+If you used {product-title} 3 and want to try {product-title} 4, you need to understand how different {product-title} 4 is. {product-title} 4 weaves the Operators that package, deploy, and manage Kubernetes applications and the operating system that the platform runs on, {op-system-first}, together seamlessly. Instead of deploying machines and configuring their operating systems so that you can install {product-title} on them, the {op-system} operating system is an integral part of the {product-title} cluster. Deploying the operating system for the cluster machines as part of the installation process for {product-title}. See xref:../migrating_from_ocp_3_to_4/planning-migration-3-4.adoc#migration-comparing-ocp-3-4[Comparing {product-title} 3 and {product-title} 4].
```

Comment:
The target page has the title "Differences between OpenShift Container Platform 3 and 4". You could consider updating this cross reference description to match that:

```
[Differences between {product-title} 3 and 4]
```